static/app/components/workflowEngine/simpleTable/index.stories.tsx

```diff
@@ -71,27 +71,21 @@ export default Storybook.story('SimpleTable', story => {
         </p>
         <SimpleTableWithColumns>
           <SimpleTable.Header>
-            <SimpleTable.HeaderCell name="name" sortKey="name">
-              Name
-            </SimpleTable.HeaderCell>
-            <SimpleTable.HeaderCell name="monitors" sortKey="monitors">
-              Monitors
-            </SimpleTable.HeaderCell>
-            <SimpleTable.HeaderCell name="action" sortKey="action">
-              Action
-            </SimpleTable.HeaderCell>
-            <SimpleTable.HeaderCell name="lastTriggered" sortKey="lastTriggered">
+            <SimpleTable.HeaderCell sortKey="name">Name</SimpleTable.HeaderCell>
+            <SimpleTable.HeaderCell sortKey="monitors">Monitors</SimpleTable.HeaderCell>
+            <SimpleTable.HeaderCell sortKey="action">Action</SimpleTable.HeaderCell>
+            <SimpleTable.HeaderCell sortKey="lastTriggered">
```

Comment:
the types, and this story, are still bad because nothing listens to the `handleSortClick` callback.